### PR TITLE
Update utils.js

### PR DESCRIPTION
**Pull request by Edebo**
issue #457 test function template for testing files

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to static food data?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] refactor
- [ ] documentation
- [ ] other

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

Are tests included?
- [ ] yes (*bugfixes and features will not be merged without tests*)
- [ ] no

Breaking Changes?
- [ ] yes (*breaking changes will not be merged unless absolutely necessary*)
- [ ] no

List any relevant issue numbers:

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -10,13 +10,22 @@ const __generateDate = () => {
   return dayjs().toDate()
 }
 
-const testFile= (file) => {
-
+const testFile = (file) => {
```
Comment (atherdon) [inline src/utils.js]: Can we have a more detailed name for function - because right now it's not very clear of what it's doing

Comment (Edebo) [reply src/utils.js]: cant think of one . it should expect a json file passed to it not to be empty

Comment (atherdon) [reply src/utils.js]: ok, this is a more better explanation than testFile

**Comment by atherdon** [comment]
should i merge this pr and you'll change that name in next time or wait?

**Comment by Edebo** [comment]
pls give me a name and i will change it now

**Comment by atherdon** [comment]
"json file passed to it not to be empty" let's be this

**Comment by atherdon** [comment]
perfect. merging

**Pull request merged**